cfmt.go

```diff
@@ -1,211 +1,217 @@
 }
 
 // Ferror prints red colored text in manner of fmt.Fprint
 func Ferror(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgRed).Fprint(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Red(fmt.Sprint(a...))))
 	return
 }
 
 // Error prints red colored text in manner of fmt.Print
 func Error(a ...interface{}) (n int, err error) {
-	n, err = Ferror(color.Output, a...)
+	n, err = Ferror(output, a...)
 	return
 }
 
 // Serror returns red colored string in manner of fmt.Sprint
 func Serror(a ...interface{}) string {
-	return color.New(color.FgRed).Sprint(a...)
+	return fmt.Sprint(aurora.Red(fmt.Sprint(a...)))
 }
 
 // Fsuccessln prints green colored text in manner of fmt.Fprintln
 func Fsuccessln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgGreen).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Green(fmt.Sprintln(a...))))
 	return
 }
 
 // Successln prints green colored text in manner of fmt.Println
 func Successln(a ...interface{}) (n int, err error) {
-	n, err = Fsuccessln(color.Output, a...)
+	n, err = Fsuccessln(output, a...)
 	return
 }
 
 // Ssuccessln returns green colored string in manner of fmt.Sprintln
 func Ssuccessln(a ...interface{}) string {
-	return color.New(color.FgGreen).Sprintln(a...)
+	return fmt.Sprint(aurora.Green(fmt.Sprintln(a...)))
 }
 
 // Finfoln prints cyan colored text in manner of fmt.Fprintln
 func Finfoln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgCyan).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, aurora.Cyan(fmt.Sprintln(a...)))
 	return
 }
 
 // Infoln prints cyan colored text in manner of fmt.Println
 func Infoln(a ...interface{}) (n int, err error) {
-	n, err = Finfoln(color.Output, a...)
+	n, err = Finfoln(output, a...)
 	return
 }
 
 // Sinfoln returns cyan colored string in manner of fmt.Sprintln
 func Sinfoln(a ...interface{}) string {
-	return color.New(color.FgCyan).Sprintln(a...)
+	return fmt.Sprint(aurora.Cyan(fmt.Sprintln(a...)))
 }
 
 // Fwarningln prints yellow colored text in manner of fmt.Fprintln
 func Fwarningln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgYellow).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, fmt.Sprint(aurora.Brown(fmt.Sprintln(a...))))
 	return
 }
 
 // Warningln prints yellow colored text in manner of fmt.Println
 func Warningln(a ...interface{}) (n int, err error) {
-	n, err = Fwarningln(color.Output, a...)
+	n, err = Fwarningln(output, a...)
 	return
 }
 
 // Swarningln returns yellow colored string in manner of fmt.Sprintln
 func Swarningln(a ...interface{}) string {
-	return color.New(color.FgYellow).Sprintln(a...)
+	return fmt.Sprint(aurora.Brown(fmt.Sprintln(a...)))
 }
 
 // Ferrorln prints red colored text in manner of fmt.Fprintln
 func Ferrorln(w io.Writer, a ...interface{}) (n int, err error) {
-	n, err = color.New(color.FgRed).Fprintln(w, a...)
+	n, err = fmt.Fprint(w, aurora.Red(fmt.Sprintln(a...)))
```

Comment:
Please change

`fmt.Fprint(w, aurora.Red(fmt.Sprintln(a...)))`

to

`fmt.Fprintln(w, aurora.Red(fmt.Sprint(a...)))`